elements/lisk-chain/test/unit/block_header.spec.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright © 2019 Lisk Foundation
```

Comment:
if its a new file then update license
```suggestion
 * Copyright © 2021 Lisk Foundation
```